mne/viz/evoked.py

```diff
@@ -673,9 +680,27 @@ def plot_evoked(evoked, picks=None, exclude='bads', unit=True, show=True,
         The axes to plot to. If list, the list must be a list of Axes of
         the same length as the number of channel types. If instance of
         Axes, there must be only one channel type plotted.
-    gfp : bool | 'only'
-        Plot GFP in green if True or "only". If "only", then the individual
-        channel traces will not be shown.
+    gfp : bool | 'only' | 'power' | 'power-only'
```

Comment:
The values for this parameter are pretty confusing. It seems like this parameter actually controls two more or less independent options:

1. Plot GFP (aka RMS) or GPF**2 (aka MS) or don't plot at all
2. If something is plotted in option 1, either plot individual traces (`True`) or not (`False`)

Also, it should be RMS and not RMS power - this wording is used in several locations.

Reply:
> The values for this parameter are pretty confusing. It seems like this parameter actually controls two more or less independent options:

Yes, that's why I wanted to introduce a second param, but I thought @larsoner, you, and @sappelhoff thought this was not a good idea (per https://github.com/mne-tools/mne-python/pull/8775#issuecomment-766951111)?

Reply:
> GPF**2 (aka MS) 

This is something we need to discuss, because there's again 2 options here.

The previous behavior (which was erroneously called "GFP") was to calculate the RMS. What **you** proposed -- squaring the actual GFP -- would **also** be **some kind** of RMS, but not the same as the one we had before. Because it would now be based on the avg-referenced signal (because GFP is based on the avg-referenced signal), while the previous approach would **not** re-reference anything.

Reply:
Yes, but at that time I did not know that there would be two more options involving "only"...

Reply:
My original proposal can be found in https://github.com/mne-tools/mne-python/pull/8775#issuecomment-766947428

Reply:
@cbrnr Please note that in `master`, we already have `gfp=True` and `gfp='only'` to select between GFP + channels and GFP only. So adding `'rms'` and `'rms-only'` seems natural, although I do agree this is not a very clean API.